### DISCUSSION TITLE
Disable MotionSensor by rule

### DISCUSSION THREAD

**Discussion by bananajoe86** [arendst/Tasmota#13758] [Q&A] [answered]
Hey guys, I need your help.
I would like to implement the following task. In the basement I have a PIR sensor that automatically controls my light on / off. There is also a switch that switches the light. It's going up to here. Now I want to deactivate the PIR sensor (Rule1) with a double press. The rule should be started again with the next double press. Here's what I have so far.

rule1
ON Power2#state=1 DO  Power1 1 ENDON ON Power2#state=0 DO  Power1 0 ENDON //(1) Power2 is the Motion and switch the light Power1 on/off

rule2
ON Switch3#state=3 DO Backlog Rule1 0 ; RuleTimer1 600 ; Power  1 ENDON  //(2)double press switch rule1 off
ON Rules#Timer=1 DO Rule1 1 ENDON                                                              //(3)after 10min rule 1 goes on
ON Power3#State=0 DO  Power1  TOGGLE ENDON                                            //(4)switch toogle light 
ON Power3#State=1 DO  Power1  TOGGLE ENDON                                            //(5)switch toogle light 
ON Power1#State=0 DO Rule1 1 ENDON                                                           //(6)light off / rule1 on (motion activate)

The last rule (6) only work, if I switch Power1 manuall off. It doesn´t work, if Power1 switched off after rule (4/5)
Has anybody a better idea to implement it or can explain me, why (6) doesn´t work with a rule ahead?

Thx at all

**Comment by ascillato** [comment, marked as answer]
You can use the rules of the PIR examples of the docs:

https://tasmota.github.io/docs/Rules/#rule_5
